src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -38,6 +35,10 @@ const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tag
         );
     };
 
+    const HandleLinkClick = (e: React.MouseEvent) => {
```

Comment:
```suggestion
    const handleLinkClick: React.MouseEventHandler<HTMLAnchorElement> = (e) => {
```

Reply:
Have you been looking too much at the backend?;)

Reply:
:D